composer.json

```diff
@@ -30,10 +30,10 @@
         "php": "~7.0",
         "deployer/phar-update": "~2.0",
         "pimple/pimple": "~3.0",
-        "symfony/console": "~2.7|~3.0",
-        "symfony/finder": "~2.7|~3.0",
-        "symfony/process": "~2.7|~3.0",
-        "symfony/yaml": "~2.7|~3.0"
+        "symfony/console": "~2.7 || ~3.0 || ~4.0",
```

Comment:
Version specification is wrong.

x "~2.7 || ~3.0 || ~4.0"
o "~2.7|~3.0|~4.0"

Reply:
both are correct and work.